AdminWebsite/AdminWebsite.IntegrationTests/Helper/MockBackchannel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AdminWebsite.IntegrationTests.Helper
+{
+    public class MockBackchannel : HttpMessageHandler
+    {
+        protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+        {
+            if (request.RequestUri.AbsoluteUri.Equals("https://inmemory.microsoft.com/common/.well-known/openid-configuration"))
+            {
+                return await EmbeddedResourceReader.GetOpenIdConfigurationAsResponseMessage("microsoft-openid-config.json");
+            }
+
+            if (request.RequestUri.AbsoluteUri.Equals("https://inmemory.microsoft.com/common/discovery/keys"))
+            {
+                return await EmbeddedResourceReader.GetOpenIdConfigurationAsResponseMessage("microsoft-wellknown-keys.json");
+            }
+
+            throw new NotImplementedException();
```

Comment:
Maybe `InvalidOperationException` or a message included explaining what's not implemented? If we get in here for some reason it's probably useful to let the caller know why they've done something wrong.